flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/operator/CEPOperatorTest.java

```diff
@@ -654,6 +654,7 @@ public boolean filter(Event value) throws Exception {
 						Lists.newArrayList(startEvent2, endEvent, middleEvent3)
 				)
 		);
+		harness.close();
```

Comment:
I think we should do this in a finally block, and modify other tests to do the same.